guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -44,3 +49,10 @@ Note this is for running Foreman itself.
 Clients will remain supported.
 
 For more details and discussion, see https://community.theforeman.org/t/drop-support-for-running-on-el8-with-foreman-3-13/38083.
+
+=== Running Foreman on Debian 11 removal in Foreman 3.14
```

Comment:
I pretty much copied the title from the previous heading so it would become inconsistent.

`Deprecating` in the title is IMHO redundant because it's already in the section with deprecations and the version where it'll be removed is also useful.

And again, I dislike `{Team}` because it's `Foreman community` so it'd expand to `Foreman community encourages users`. That doesn't make any sense to me.